policies/loading-frame-default-eager.md

```diff
@@ -0,0 +1,46 @@
+loading-frame-default-eager Policy
+===========
+
+The `loading-frame-default-eager` policy-controlled feature can be used for a main frame or a nested
+`<iframe>` to overwrite the default or assigned value of the newly
+introduced `loading` [attribute](https://github.com/whatwg/html/pull/3752) for `<iframe>` and `<img>`.
```

Comment:
... and `<img>`? Weren't we discussing a separate policy for images?

Reply:
This part defines what `loading` attribute is; which is related to both `<img>` and `<iframe>`.

Reply:
We shouldn't need to explain `loading` -- this feels like it should be the introduction to `loading-frame-default-eager`, so I parsed it incorrectly as "feature can be used ... to overwrite the default value ... for `<iframe>` and `<img>`"

Reply:
Newest version does not name `<img>`.